contracts/services/PayoutManager.sol

```diff
@@ -36,7 +15,8 @@ contract PayoutManager {
     uint256 private currentPayoutId = 0; // current payout ID, incremental - if some payout exists, it will always be smaller than this value
     mapping(uint256 => Payout) private payoutsById;
     mapping(address => uint256[]) private payoutsByAssetAddress;
-    mapping(uint256 => mapping(address => bool)) private payoutClaims;
+    mapping(address => uint256[]) private payoutsByOwnerAddress;
+    mapping(uint256 => mapping(address => uint256)) private payoutClaims;
 
     //------------------------
     //  EVENTS
```

Comment:
I would also move events to the interface because events report state-changing functions that are defined in the interface.